src/UI/Component/Item/AppointmentItem.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Item;
+
+/**
+ * Interface AppointmentItem
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface AppointmentItem extends Item {
+	/**
+	 * Get the starting point of the appointment.
+	 * @return \ilDateTime
+	 */
+	public function getFrom();
+
+	/**
+	 * Get the ending point of the appointment.
+	 * @return \ilDateTime
+	 */
+	public function getEnd();
```

Comment:
I think we might currently not need an appointment/blog/forum list at all. These things are only part of the originally accepted interface. But in this proposal the Standard List comes with (optional) features that fit the requirements of the appointment list visualisation. (See my initial comment)